benchmarking/smartseq2/RunBenchmarkingAnalysis.wdl

```diff
@@ -0,0 +1,55 @@
+import "BenchmarkingTasks.wdl" as analysis
+
+workflow RunBenchmarkingAnalysis {
+  String base_datafile
+  String updated_datafile
+  String output_name
+  String gtf_file
+  String metadata_file
+  String metadata_keys
+  String metKeys
+  String base_metrics
+  String updated_metrics
+  Int    npcs
+
+  meta {
+    description: "Process SmartSeq2 Benchmarking pipeline"
```

Comment:
I could use more information in this description. Could you summarize what the workflow does in a little more detail?